Adds site validation script for testing new installations (#45)
RhinoHealth/user-resources: sandbox/site-testing/README.md

> # Site Validation Script

Reviewer: Since this example is with financial data, and the majority of our clients are healthcare focused, I think we should call this something like "site-testing-financial" and have a similar "site-testing-health" with healthcare data. For example - I assume Lilly wouldn't be excited about validating their installation with financial data.
WDYT?

Author: Makes sense. I'll rename the directory

> The goal of this project is to validate the installation of a new client site. It uses the Rhino SDK to simulate the workflow of a standard project.

Reviewer: ```suggestion
The goal of this project is to validate the installation of a new Rhino Client. It uses the Rhino SDK to simulate the workflow of a standard project.
```

In this directory you will find two directories and the notebook to run the testing

> - `./data/` - a directory containing the data to upload into the cloud mounted storage

Reviewer: ```suggestion
- `./data/` - a directory containing the data to make accessible to your Rhino Client
```

- `./model/` - a directory containing an NVFlare model for testing Autocontainer NVFlare functionality
- `./site-testing.ipynb` - the Jupyter Notebook to run the process

This page will document the steps needed to perform the site validation.

## Preparing the Project

> 1. Copy the `./data/credit_risk_dataset.csv` file to the location you wish to use for data uploads. This can be either on the Rhino Client (`/rhino_health/`) or on an external data store (e.g. S3, GCS)

Reviewer: ```suggestion
1. Copy the `./data/credit_risk_dataset.csv` file to the location you wish to use for data accessible to the Rhino Client. This can be either on the Rhino Client (`/rhino_data/`) or on a client-mounted data store (e.g. S3, GCS).
```

> 2. Gather the login credentials for a Rhino Cloud user

Reviewer: ```suggestion
2. Make sure you have your login credentials to the Rhino FCP
```

3. Update the `./site-testing.ipynb` file with the values in step 2.

>    - `USERNAME` - The username of the Rhino Cloud user

Reviewer: ```suggestion
   - `USERNAME` - The username (email) of your Rhino FCP user
```

>    - `DATASTORE_PATH` - The path to where you copied the `./data` folder, either locally or on cloud storage

Reviewer: ```suggestion
   - `CLIENT_DATA_PATH` - The path to where you copied the contents of the `./data` folder, either within `/rhino_data/` or the mount point for client-mounted storage (e.g. `/rhino_data/external/my_storage_bucket`)
```
